app/controllers/api/v1x2/stageaction/result.html.erb

```diff
@@ -61,14 +61,22 @@
                     </span>Denied</span>
                   </span>
                 </span>
-              <% else %>
+              <% elsif @order[:status] == "error" %>
                 <span class="pf-c-label pf-m-red pf-m-outline">
                   <span class="pf-c-label__content">
                     <span class="pf-c-label__icon">
                       <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
                     </span>Error</span>
                   </span>
                 </span>
+              <% else %>
+                <span class="pf-c-label pf-m-outline">
+                  <span class="pf-c-label__content">
+                    <span class="pf-c-label__icon">
+                      <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
+                    </span>Undecided</span>
```

Comment:
If the styles for `Error` and `Undecided` are same, we should combine them together:
```
              <% else %>
                <span class="pf-c-label pf-m-red pf-m-outline">
                  <span class="pf-c-label__content">
                    <span class="pf-c-label__icon">
                      <i class="fas fa-fw fa-info-circle" aria-hidden="true"></i>
                    </span> <% @order[:status].capitalize %></span>
                  </span>
                </span>
```

Reply:
@hsong-rh They are different - Error is red, undecided is gray.

Reply:
I see, then let's use `</span> <% @order[:status].capitalize %></span>` in the `else`. It will handle another possible order status value `cancelled`, just in case.